int16 delta_millis
int32 left_ticks
int32 right_ticks
time timestamp